content/news/playtest-20160424.md

```diff
@@ -0,0 +1,77 @@
+We are happy to present to you the second playtest in this release cycle!
+
+Many users &ndash; indeed more than we anticipated (over 2000!) &ndash; have tried out the first playtest and gave us a lot of feedback. Almost inevitably, a number of bugs have also been reported (thanks to all the testers who took their time to do so!), many of them quite hard to track down.
+
+We have now fixed all of those issues, including:
+
+* Crashes when opening the in-game menu on some maps (Fort Lonestar, for example).
+* Broken left click orders.
+* Way too many infantry units coming out of sold buildings.
+* Submarines not being targetable.
+* Stealth units being targetable even when stealthed under some circumstances.
+
+Additionally, the Red Alert mod has seen more refinements to its map pool: a few lower quality maps have been removed, while some new tournament-grade maps have been added.
+
+As usual, the complete list of changes can be seen [on our wiki](http://github.com/OpenRA/OpenRA/wiki/Changelog/HASHHASHASH).
+
+We promised in the last news post to detail the new map format and other map-related changes here. Skip to below the break for the full scoop.
+
+Installers for all supported operating systems are available from [the download page](/downloads). Of course we are still interested to hear about any bugs and issues you come across, so please don't hesitate to report them [on our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments.
+
+At this point we'd like to reiterate a topic already mentioned in the previous news post. In order to get a better understanding of the composition of our user base and the way our software is being used, we decided to add an anonymous, opt-out data gathering facility to OpenRA. When enabled, the game will send us a few technical details about your system (see below). As already mentioned, this data is sent to us anonymously, using a randomly generated ID. If you are uncomfortable with this, you can disable this functionality by unchecking the "Send System Information" checkbox in the advanced settings dialog, or by starting the game using the `Debug.SendSystemInformation=false` argument.
```

Comment:
Could you mention changing your settings.yaml manually, too?
I reckon most Windows / OS X users are launching this with a double-click and don't want to (or wouldn't know how to) pass this argument.